CONTRIBUTING.md

```diff
@@ -61,31 +61,91 @@ You can also clean up generated files:
 
 `yarn clean`
 
+## Releasing packages
 
-## Pull Requests, versioning and releasing
+Automated releases to the npm public registry are triggered for all pull requests containing modifications to one or more npm packages (found in the `/packages/` directory). The information required to determine the version update for each release is taken from the title and content of the pull request.
 
-To deploy Kaizen Component Library changes, your **Pull Request (PR) title must follow the [Conventional Commits format](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)** and “Semantic Versioning” (SemVer):
+### Release workflow
 
-> Given a version number MAJOR.MINOR.PATCH, increment the:
-    MAJOR version when you make incompatible API changes,
-    MINOR version when you add functionality in a backwards compatible manner, and
-    PATCH version when you make backwards compatible bug fixes.
+To release a new package version, create a pull request which **modifies ONLY the package you wish to release**, and satisfies **ONE** of the following:
+
+- If the branch contains multiple commits, the pull request must have a [conventional title](#conventional-commit)
+- If the branch contains only a single commit...
+  - ... the commit itself must have a [conventional commit message](#conventional-commit), and...
+  - ... (for consistency) the PR title must match that commit message
```

Comment:
Wonder if this could be simplified to something like:
```
To release a new package version, create a pull request which **modifies ONLY the package you wish to release**.

- Make sure your PR has a [conventional title](#conventional-commit)
- If there is only **one commit** in your PR, make sure that commit message matches the PR title
```
Will edit.